vpr/src/device/rr_graph_builder.h

```diff
@@ -235,6 +244,11 @@ class RRGraphBuilder {
     /* Fast look-up for rr nodes */
     RRSpatialLookup node_lookup_;
 
+    vtr::vector<RRSegmentId, t_segment_inf> rr_segments_;
```

Comment:
Please add comments to these internal data. Refer to the code comments here: 
https://github.com/verilog-to-routing/vtr-verilog-to-routing/blob/116f30cb8e8f791fb00816b1f8df0f5772bc84fa/vpr/src/device/rr_graph_obj.h#L810-L824